lib-py/casmutils/xtal/_xtal/xtal-py.cxx

```diff
@@ -72,21 +91,22 @@ PYBIND11_MODULE(_xtal, m)
 
     {
         using namespace wrappy::Site;
+        typedef xtal::Site xSite;
```

Comment:
I think this typedef is more harm than good. It was present for `xCoord` because the functions were getting super long, but with the `overload_cast` you figured out, you probably don't need it there either anymore.